scripts/run_benchmark/run_full_seqeracloud.sh

```diff
@@ -10,7 +10,7 @@ set -e
 
 # generate a unique id
 RUN_ID="run_$(date +%Y-%m-%d_%H-%M-%S)"
-publish_dir="s3://openproblems-data/resources/task_denoising/results/${RUN_ID}"
+publish_dir="s3://openproblems-data/resources/results/${RUN_ID}"
```

Comment:
I guess you meant to remove the task name from here?

Reply:
No, the results need to be published in the task specific subdirs ^^ Coincidentally I had make the same changes but then reverted them because I realised that this in fact made sense